app/Listeners/SendTelegramNewArticleNotification.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Listeners;
+
+use App\Events\ArticleWasCreated;
+use App\Notifications\ArticleSubmitted;
+use Illuminate\Notifications\AnonymousNotifiable;
+
+final class SendTelegramNewArticleNotification
+{
+    public function handle(ArticleWasCreated $event): void
+    {
+        (new AnonymousNotifiable())->notify(new ArticleSubmitted($event->article));
```

Comment:
Can we typehint `AnonymousNotifiable` in the constructor as we do in when [posting to Twitter](https://github.com/laravelio/laravel.io/blob/main/app/Console/Commands/PostArticleToTwitter.php#L16)

Reply:
Done